xbmc/addons/ContextItemAddon.cpp

```diff
@@ -63,6 +60,13 @@ CContextItemAddon::CContextItemAddon(const cp_extension_t *ext)
   }
 }
 
+std::string CContextItemAddon::GetLabel()
+{
+  if (StringUtils::IsNaturalNumber(m_label))
+    return GetString(boost::lexical_cast<int>(m_label.c_str()));
```

Comment:
how often is this method called when the content menu is up. For every frame or is it cached somewhere? If not it might make sense to cache the translated string in m_label like it was the case before?